Whitepapers.md

```diff
@@ -1,24 +1,37 @@
+| Qtum                 | Smart-Contract Value-Transfer Protocols on a Distributed Mobile Application Platform | https://qtum.org/uploads/files/a2772efe4dc8ed1100319c6480195fb1.pdf                                               | https://qtum.org/                  | Patrick Dai, Neil Mahi, Jordan Earls, Alex Norta                                                         |
+| SALT                 | SALT: Blockchain-Backed Loans                                                        | https://membership.saltlending.com/files/abstract.pdf                                                             | https://www.saltlending.com/       |                                                                                                          |
+| Steem                | Steem: An incentivized, blockchain-based, public content platform                    | https://steem.io/SteemWhitePaper.pdf                                                                              | https://steem.io/                  |                                                                                                          |
+| Stellar Lumens       | The Stellar Consensus Protocol: A Federated Model for Internet-level Consensus       | https://www.stellar.org/papers/stellar-consensus-protocol.pdf                                                     | https://www.stellar.org/           | David Mazières                                                                                           |
+| Stratis              | Stratis: Blockchain Solutions                                                        | https://stratisplatform.com/files/Stratis_Whitepaper.pdf                                                          | https://stratisplatform.com/       | Chris Trew, Guy Brandon and Nicolas Dorier                                                               |
+| TenX                 | TenX: Payment Platform Whitepaper                                                    | https://www.tenx.tech/whitepaper/tenx_whitepaper_final.pdf                                                        | https://www.tenx.tech/             |                                                                                                          |
+| Tether               | Tether: Fiat currencies on the Bitcoin blockchain                                    | http://www.the-blockchain.com/docs/Tether%20Whitepaper.pdf                                                        | https://tether.to/                 |                                                                                                          |
+| TRON                 | TRON: Whitepaper                                                                     | https://www.dropbox.com/s/yz8ha60t4h25pwz/Tron-Whitepaper-0822-V17-EN%20%281%29.pdf?dl=0                          | https://tronlab.com/en.html        |                                                                                                          |
```

Comment:
We need to find another source for this whitepaper, Dropbox isn’t going to work out.